src/templates/XmlRenderer/types_data.tsx

```diff
@@ -0,0 +1,131 @@
+// Generated by https://quicktype.io
+
+export interface XmlRendererInterface {
```

Comment:
filename can follow camelcase (same as the other templates) instead of underscore?

Reply:
Done.